### PR TITLE
Combined docs: Add README and Tripal JBrowse Integration docs.

### PR DESCRIPTION
**Pull request by laceysanderson**
This PR adds a README to the repo at the top level. This readme is for the new combined package and provides a quickstart and link to new documentation.

I've also added a `doc` directory at the top level containing documentation for Tripal JBrowse which you can see rendered here: https://tripal-jbrowse.readthedocs.io/en/combined-docs/

### CODE REVIEW

**Comment by almasaeed2010** [comment]
We probably need to add acknowledgements, license and copyright notice at the bottom of the readme or the RTD docs. Where do you think it'll be appropriate? Otherwise, this looks perfect to me. I'll submit a new PR once this is merged to add the tripal_jbrowse_mgmt documentation (we already have them on the old repo). 

On a side note, @jwest60  contributed multiple features to the mgmt module but his commit history got removed with the migration so possibly we can add his name too to acknowledge his contribution?

Thanks!

**Comment by bradfordcondon** [comment]
if you really want you ammend the commit that erased the history to include him as a co-author:  https://help.github.com/articles/creating-a-commit-with-multiple-authors/

**Comment by laceysanderson** [comment]
@almasaeed2010 do you want to add the additional sections to the bottom of the README? That way they are worded the way you prefer. You can commit directly to this branch. I'll add my side of things once you add yours, to make sure we have consistent wording :-) Alternatively, you can add your words in a comment here and I could commit if that's faster for you :-)

As far as attribution to @jwest60, we can definitely add his name in the acknowledgements section! However, ideally we would fix the commit history to ensure he has complete recognition.

**Comment by almasaeed2010** [comment]
I am working on adding @jwest60 as a co-author as @bradfordcondon suggested. 

I can add the copyright section directly to here as well.

**Comment by almasaeed2010** [comment]
Alright all good to go. @laceysanderson can merge after reviewing the new additions.

Thanks!

File: docs/_static/theme_overrides.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}
+.wy-side-nav-search input[type="text"] {
+  border-color: black;
+}
+.wy-nav-side {
+  color: #CCCCCC !important;
+  background: #2D2D34 !important;
+  background-image: url("hexagon_pattern.png") !important;
+  background-repeat: repeat !important;
+  background: -webkit-linear-gradient(left, $light-grey , $gray-base) !important;
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Hi @laceysanderson. I just noticed this little issue. This line uses SCSS variables. This is invalid in css. Do you know the actual values for the variables `$light-grey` and `$grey-base`?

Thanks!

File: docs/_static/theme_overrides.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  background: linear-gradient(to right, $light-grey , $gray-base) !important;
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Same issue as above

File: docs/_static/theme_overrides.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+}
+.wy-menu-vertical li.current a:hover,
+  .wy-menu-vertical li.toctree-l1.current > a:hover,
+  .wy-menu-vertical li.toctree-l2.current > a:hover,
+  .wy-menu-vertical li.toctree-l2.current li.toctree-l3 > a:hover,
+  .wy-menu-vertical li.toctree-l3.current li.toctree-l4 > a:hover {
+    color: white;
+    border-color: black;
+    font-weight: bold;
+    background-color: rgba(0,0,0);
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Missing the alpha value.
```suggestion
    background-color: rgba(0,0,0, 1);
```

File: docs/_static/theme_overrides.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  .rst-content .note, 
+  .rst-content .wy-alert-info.attention, 
+  .rst-content .wy-alert-info.caution, 
+  .rst-content .wy-alert-info.danger, 
+  .rst-content .wy-alert-info.error, 
+  .rst-content .wy-alert-info.hint, 
+  .rst-content .wy-alert-info.important, 
+  .rst-content .wy-alert-info.tip, 
+  .rst-content .wy-alert-info.warning, 
+  .rst-content .seealso, .rst-content
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Missing comma
```suggestion
  .rst-content .seealso, .rst-content,
```

File: docs/_static/theme_overrides.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  .rst-content .wy-alert-info.hint, 
+  .rst-content .wy-alert-info.important, 
+  .rst-content .wy-alert-info.tip, 
+  .rst-content .wy-alert-info.warning, 
+  .rst-content .seealso, .rst-content 
+  .wy-alert-info.admonition-todo, 
+  .rst-content .wy-alert-info.admonition {
+    background: #d9d9d9;
+}
+.wy-alert.wy-alert-info
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Missing comma
```suggestion
.wy-alert.wy-alert-info, 
```

File: docs/_static/theme_overrides.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+  .rst-content .note .admonition-title, 
+  .rst-content .wy-alert-info.attention .admonition-title, 
+  .rst-content .wy-alert-info.caution .admonition-title, 
+  .rst-content .wy-alert-info.danger .admonition-title, 
+  .rst-content .wy-alert-info.error .admonition-title, 
+  .rst-content .wy-alert-info.hint .admonition-title, 
+  .rst-content .wy-alert-info.important .admonition-title, 
+  .rst-content .wy-alert-info.tip .admonition-title, 
+  .rst-content .wy-alert-info.warning .admonition-title, 
+  .rst-content .seealso .admonition-title, .rst-content
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Missing comma
```suggestion
  .rst-content .seealso .admonition-title, .rst-content, 
```

File: docs/_static/theme_overrides.css (hunk 1 above)
Comment (laceysanderson) [reply docs/_static/theme_overrides.css]: Oops! I do not actually, we'll have to ask @shawnawsu as this is from the tripal.info site theme 😬 
screen shot 2018-12-03 at 10 26 11 am

Comment (almasaeed2010) [reply docs/_static/theme_overrides.css]: I am currently testing this css and will push changes in a few

**Comment by almasaeed2010** [comment]
Ok everything in the CSS seems to work as expected once the variables are removed!

**Review by almasaeed2010 — APPROVED** [review]

**Comment by laceysanderson** [comment]
Awesome! Thanks :-)

**Pull request merged**